Refit/MultipartItem.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refit
+{
+    public abstract class MultipartItem
+    {
+        public MultipartItem()
+        {
+
+        }
+
+        public MultipartItem(string fileName, string contentType)
+        {
+            this.FileName = fileName;
```

Comment:
FileName is technically not required as it can fallback to the parameter name (or the to-be deprecated attribute)